amethyst_renderer/src/light.rs

```diff
@@ -31,14 +30,14 @@ pub struct DirectionalLight {
     /// Color of the light in RGBA8 format.
     pub color: Rgba,
     /// Direction that the light is pointing.
-    pub direction: Vector3<f32>,
+    pub direction: [f32; 3],
```

Comment:
Wasn't the goal consistency? So could we maybe just upgrade to gfx 0.17?

Reply:
0.17 isn't on crates :(

Reply:
Although I'll add a Todo for when we do upgrade our gfx